lib/core/src/lib/styles/_reference-variables.scss

```diff
@@ -17,3 +17,9 @@ $adf-ref-line-height: 40px;
 $adf-ref-margin-right: 8px;
 $adf-ref-tab-bottom-line-default: unset;
 $adf-ref-tab-bottom-line-active: unset;
+$adf-ref-metadata-property-panel-border-color: rgba(0, 0, 0, 0.12);
+$adf-ref-metadata-buttons-background-color: rgba(33, 33, 33, 0.05);
+$adf-ref-metadata-action-button-clear-color: #212328b2;
```

Comment:
We should not use fixed colours here, if someone is going to change completely theme, we still going to have this colour, which can not fit with the our theme.

Reply:
Updated. Please check.
@BSekula @DenysVuika